prediction_prophet/deployment/models.py

```diff
@@ -67,7 +67,7 @@ def calculate_bet_amount(self, answer: bool, market: AgentMarket) -> BetAmount:
         else:
             should_not_happen(f"Unknown market type: {market}")
         if amount > max_bet_amount:
-            logger.warning(f"Would need at least {amount} {market.currency} to be profitable, betting only {market.get_tiny_bet_amount()} for benchmark purposes.")
+            logger.warning(f"Calculated amount {amount} {market.currency} is exceeding our limit {max_bet_amount=}, betting only {market.get_tiny_bet_amount()} for benchmark purposes.")
```

Comment:
Is the `=` in `{max_bet_amount=}` intentional or a typo? Not sure if this is a feature of Python I'm unfamiliar with.

Reply:
It's a feature of f-strings that prints the variable alongside its value 😃 
```
>>> bar = 10.3
>>> print(f"foo {bar=} baz")
foo bar=10.3 baz
```

Reply:
It's a feature of f-strings. It will print the variable name automatically:

```
>>> x = 10
>>> print(f"{x=}")
x=10
```

Reply:
Appreciate the explanation!